docs/index.rst

```diff
@@ -1,10 +1,56 @@
-Welcome to the Software Factory documentation
-*********************************************
+Welcome to the PELUX Baseline Software Factory
+**********************************************
 Revision: |release|
 
 -------------------
 
-The Software Factory documentation is the central space where we collect all useful information on how to work within the project.
+What is the Software Factory?
+-----------------------------
+The Software Factory documentation contains processes, instructions and how-tos
+for how to work with PELUX, as well as instructions for how to create a
+standalone project based on PELUX.
+
+What is PELUX?
+--------------
+From pelux.io_:
+
+    *PELUX is a Linux based platform for your automotive infotainment project.
+    The PELUX software architecture is optimized to secure the shortest path
+    from silicon to pixel and thereby provide optimal performance to deliver a
+    stunning infotainment solution. It’s based on Yocto and offers an improved
+    developer experience.*
+
+
+Why PELUX?
+^^^^^^^^^^
+PELUX is a base platform for building automotive infotainment systems (IVI
+systems for short). IVI systems are usually responsible for everything shown in
+the dashboard of the car as well as the head-unit, which is usually placed in or
+above the center stack where one normally had just a radio back in the days.
+
+IVI systems are obviously put into cars by its car maker. However, most car
+makers don't develop the software for these systems in-house, but they contract
+a large software supplier. These software suppliers usually provide both
+hardware and software as a bundle.
+
+With such a bundle, the software usually will only work on the specific
+hardware, and the only one who the car maker can utilize for upgrades or support
+is the supplier. This leads to vendor lock-in, and it is something we believe is
+bad.
```

Comment:
I would like to re-phrase this paragraph like this:
"When bundled in this way, the software is usually adapted to the hardware in such a way that it can not be used with any other hardware. Usually this also means that the car maker will need to use the supplier of the bundle for upgrades or support." + last sentence

Reply:
Sure thing!